src/features/defi/components/Deposit/Deposit.tsx

```diff
@@ -115,7 +115,10 @@ export const Deposit = ({
   }
 
   const handlePercentClick = (percent: number) => {
```

Comment:
callback?

Reply:
Added in 63c2d55575c277539d9aab81b0905470cc8bf2fe